app/views/content/train-to-be-a-teacher/if-you-dont-have-a-degree.md

```diff
@@ -35,10 +34,13 @@ Qualifications vary depending on the course. For example, you could get QTS with
 
 [Find out more about the qualifications you need to be a teacher in England](/is-teaching-right-for-me/qualifications-you-need-to-teach).
 
-As part of selecting your degree course, you will need to decide if you want to train to teach at a primary or secondary level. [Learn about deciding who to teach](/is-teaching-right-for-me/who-do-you-want-to-teach).
-
 <div class="inset">
 
 <p>If you're a non-UK citizen who wants to study in England, you can visit the <a href="https://www.ukcisa.org.uk/">UK Council for International Student Affairs</a> for information about studying at an English university.</p>
 
 </div>
```

Comment:
The Find component on this page links to UCAS for undergrad degrees - given that we can't move it away from the bottom of the page for now, I would suggest re-wording it slightly so it is clear that it relates to undergraduate degrees and not TDAs?

Reply:
Only on this page :-)